conf/quincy/rbd/4-node-cluster-with-1-client.yaml

```diff
@@ -0,0 +1,38 @@
+globals:
```

Comment:
Adding some documentation about the purpose of the cluster conf helps in maintenance and prevents conflicting changes.

Reply:
@psathyan sorry for missing, added now